docs/docs/system-administration/rbac.md

```diff
@@ -100,6 +100,7 @@ Permissions can be assigned at 3 levels: Organisation, Project, and Environment.
 | View Environment       | Can see the Environment within their account              |
 | Update Feature State   | Update the state or value for a given feature             |
 | Manage Identities      | View and update Identities                                |
+| Segment Overrides      | View and update Segment Overrides                         |
```

Comment:
The new permission is actually `Manage Segment Overrides` and it is for `Permission to manage segment overrides in the given environment`

<img width="610" alt="image" src="https://github.com/Flagsmith/flagsmith/assets/222715/5ed8671e-3409-4044-a1c9-2635f7f4656b">

Reply:
OK updated thanks!